js-extensions/spec/ResourceLoadTracker-spec.js

```diff
@@ -104,4 +104,42 @@ describe("ResourceLoadTracker.js", function () {
             done();
         });
     });
+
+    it("- test loading extension CSS from another location for a specific plugin", function (done) {
+        jsTest.onPage(function() {
+            var javaScriptExtensionInfo = require('./javaScriptExtensionInfo-02.json');
+            var jsModules = require('@jenkins-cd/js-modules');
+            var ResourceLocationResolver = require('@jenkins-cd/js-modules/js/ResourceLocationResolver');
+
+            // get the 'plugin-1' resources from another host
+            // continue to get 'plugin-2' resources from Jenkins adjuncts
+            jsModules.addResourceLocationResolver(
+                new ResourceLocationResolver(function(moduleSpec) {
+                    // 'plugin-1' is the hpiPluginId set in ./javaScriptExtensionInfo-02.json
+                    return (moduleSpec.namespace === 'plugin-1');
+                }, function(moduleSpec, srcPath) {
+                    return "http://resources.acme.com/javascript/" + srcPath;
+                })
+            );
+
+            // Initialise the load tracker with plugin extension point info.
+            ResourceLoadTracker.setExtensionPointMetadata(javaScriptExtensionInfo);
+
+            // Verify that there's no link elements on the page.
+            var cssElements = document.getElementsByTagName('link');
+            expect(cssElements.length).to.equal(0);
+
+            // Mounting ep-* should result in the CSS for both plugins being
+            // loaded ...
+            ResourceLoadTracker.onMount('ep-1');
```

Comment:
That's just a test, with `ep-1` being "extension point 1". See the other tests in the same spec.